src/components/CarouselGallery/CarouselGallery.tsx

```diff
@@ -1,23 +1,47 @@
+import Image from "next/image";
+
+import pond_anteater from "assets/artwork/ai_anteater_pond.jpg";
+import scream_anteater from "assets/artwork/anteater_scream.jpg";
+import ant_lilies from "assets/artwork/ant_lilies.jpg";
+import haring_anteaters from "assets/artwork/haring_anteaters.png";
+import mona_petr from "assets/artwork/mona_petr.jpg";
+import starry_eh from "assets/artwork/starry_EH.jpg";
+
 import styles from "./CarouselGallery.module.scss";
 
 interface CarouselGalleryProps {
 	rotating: boolean;
 }
 
+const PAINTINGS = [
+	{ image: mona_petr, aspectRatio: "4/5" },
```

Comment:
Question: How is aspectRatio used? It is not passed through the mapping, and it is not used in the CSS files.

Reply:
Currently it is not. I originally included it to preserve the original aspect ratio of the paintings rather than cropping, but that would result in awkward spacing between paintings since they are equally spaced radially, so we would need to manually adjust offset angles. We may revisit this in the future.